<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainTradeForm</class>
 <widget class="QWidget" name="TrainTradeForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lab_price">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>价格:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_price">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_qty">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_qty">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数量:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_capital_ava">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>可用资金:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_capital_ava">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_reason">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>交易理由:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_reason">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_close">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_trade">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>买入</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbt_all">
      <property name="text">
       <string>全量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbtn_qty_half">
      <property name="text">
       <string>1/2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbtn_qty_one_third">
      <property name="text">
       <string>1/3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbtn_qty_one_fifth">
      <property name="text">
       <string>1/5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lab_status">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>status</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_qty_ava_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>多空:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioBtn_long">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>多</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn_short">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>空</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioBtn_open">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn_close">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>平</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lab_qty_ava_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开平:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_qty_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>可用数量:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_qty_ava">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lab_price_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>止赢:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_stop_profit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lab_price_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>止损:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_stop_loss">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <zorder>le_stop_profit</zorder>
   <zorder>lab_price_2</zorder>
   <zorder>lab_price_2</zorder>
   <zorder>le_stop_profit</zorder>
   <zorder>lab_price_3</zorder>
   <zorder>le_stop_loss</zorder>
  </widget>
  <zorder>lab_price</zorder>
  <zorder>le_price</zorder>
  <zorder>le_qty</zorder>
  <zorder>lab_qty</zorder>
  <zorder>lab_capital_ava</zorder>
  <zorder>le_capital_ava</zorder>
  <zorder>label_reason</zorder>
  <zorder>textEdit_reason</zorder>
  <zorder>pbtn_close</zorder>
  <zorder>pbt_trade</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>lab_status</zorder>
  <zorder>lab_qty_ava_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>radioBtn_long_5</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>lab_qty_ava_3</zorder>
  <zorder>lab_qty_2</zorder>
  <zorder>le_qty_ava</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
